client/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "esnext",
+    "strict": true,
+    "skipLibCheck": true,
+    "moduleResolution": "bundler",
+    "esModuleInterop": true, // React를 ESM으로 import할 수 있게 해줘요
+
+    "jsx": "react-jsx", // JSX를 React 문법으로 변환해요
+    "jsxImportSource": "@emotion/react",
+    "baseUrl": "."
+  },
+  "include": ["./**/*.ts", "src/main.tsx", "global.d.ts"],
```

```suggestion
  "include": ["src", "global.d.ts"],
```

전체적으로 src 디렉토리 안의 파일들과 루트의 global.d.ts만 포함하면 충분할 것 같아서 이렇게 제안드려봅니다! 
유지보수 측면에서도 일일이 명시할 필요가 없어서 더 간결해질 수 있을 것 같은데, 익셉은 어떻게 생각하시나요??

---

지금 생각해보니 './**/*.ts' 라는 범위가 너무 광범위 한 것 같네요. 
차라리 세라 말씀대로 src로 명확하게 지정해주는 것이 좋아보입니다. 

3c3e4acec24933b9d3d23737d40f58386010f22f